packages/desktop-client/src/components/Titlebar.tsx

```diff
@@ -41,9 +42,10 @@ import useSheetValue from './spreadsheet/useSheetValue';
 import { ThemeSelector } from './ThemeSelector';
 import { Tooltip } from './tooltips';
 
+type Listener = (msg: string, newBudgetType?: LocalPrefs['budgetType']) => void;
 export type TitlebarContextValue = {
-  sendEvent: (msg: string) => void;
-  subscribe: (listener) => () => void;
+  sendEvent: (msg: string, newBudgetType?: LocalPrefs['budgetType']) => void;
```

Comment:
We should make the 2nd parameter here not specific to budget type as we might introduce different message types in the future.  We should instead convert the `msg` parameter to a structured object with a `type` property similar to how it is with redux actions e.g.

```json
{
  type: 'budget/switch-type',
  budgetType: newBudgetType
}
```

Reply:
Good idea. I changed that